integreat_cms/cms/views/bulk_action_views.py

```diff
@@ -169,20 +169,23 @@ def post(self, request, *args, **kwargs):
         :return: The redirect
         :rtype: ~django.http.HttpResponseRedirect
         """
-        if not settings.SUMM_AI_ENABLED or not request.region.summ_ai_enabled:
-            if not settings.SUMM_AI_ENABLED:
-                logger.warning("SUMM.AI globally disabled")
-            if not request.region.summ_ai_enabled:
-                logger.warning("SUMM.AI disabled in %r", request.region)
-            messages.error(request, _("Translations to Easy German are disabled"))
-            return super().post(request, *args, **kwargs)
-        # Collect the corresponding objects
-        logger.info(
-            "%r started SUMM.AI translation for: %r", request.user, self.get_queryset()
-        )
-        api_client = SummAiApiClient(request, self.form)
-        api_client.translate_queryset(self.get_queryset())
-        # Let the base view handle the redirect
+        if request.user.is_staff:
+            if not settings.SUMM_AI_ENABLED or not request.region.summ_ai_enabled:
+                if not settings.SUMM_AI_ENABLED:
+                    logger.warning("SUMM.AI globally disabled")
+                if not request.region.summ_ai_enabled:
+                    logger.warning("SUMM.AI disabled in %r", request.region)
+                messages.error(request, _("Translations to Easy German are disabled"))
+                return super().post(request, *args, **kwargs)
+            # Collect the corresponding objects
+            logger.info(
+                "%r started SUMM.AI translation for: %r",
+                request.user,
+                self.get_queryset(),
+            )
+            api_client = SummAiApiClient(request, self.form)
+            api_client.translate_queryset(self.get_queryset())
+            # Let the base view handle the redirect
```

Comment:
```suggestion
        if not request.user.is_staff:
            raise PermissionDenied(
                f"Only staff users have the permission to bulk translate pages via SUMM.AI"
            )
        if not settings.SUMM_AI_ENABLED or not request.region.summ_ai_enabled:
            if not settings.SUMM_AI_ENABLED:
                logger.warning("SUMM.AI globally disabled")
            if not request.region.summ_ai_enabled:
                logger.warning("SUMM.AI disabled in %r", request.region)
            messages.error(request, _("Translations to Easy German are disabled"))
            return super().post(request, *args, **kwargs)
        # Collect the corresponding objects
        logger.info(
            "%r started SUMM.AI translation for: %r",
            request.user,
            self.get_queryset(),
        )
        api_client = SummAiApiClient(request, self.form)
        api_client.translate_queryset(self.get_queryset())
        # Let the base view handle the redirect
```
In my opinion, we should always raise errors when something is not supposed to happen - just silently doing nothing makes debugging very hard when something is not working.
So in this case, raising a permission denied error is the way to go I think.
(requires `from django.core.exceptions import PermissionDenied`)